docs/source/migration.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'Migrating from the Apollo CLI'
+sidebar_title: 'Migrating from the Apollo CLI'
+---
+
+The [Apollo CLI](https://github.com/apollographql/apollo-tooling) is Apollo's previous CLI tool for managing graphs with [Apollo Studio's graph registry](https://www.apollographql.com/docs/intro/platform/#3-manage-your-graph-with-apollo-studio).
+
+This guide will help you migrate from the Apollo CLI to Rover, providing a guide for many of the key differences in Rover and the Apollo CLI, along with examples of the most common workflows from the Apollo CLI.
+
+If you haven't already read the [Conventions](./essentials) doc, start there, since this doc builds off it and assumes you have already read it. It will also be helpful to [install](./getting-started) and [configure](./configuring) Rover before reading through this guide.
+
+> **Rover is in active, rapid development.** Rover commands are subject to breaking changes without notice. Until the release of version 0.1.0 or higher, we do not yet recommend integrating Rover in critical production systems.
+>
+> For details, see [Public preview](./#public-preview).
+
+## Configuring
+
+Rover's approach to [configuration](./configuring) is much more granular and less abstracted than the Apollo CLI. Most configuration options in Rover are configured as flags on a command, rather than items in a config file. The only config files that Rover uses are hidden config files created by the `config auth` command and another as input to [`supergraph compose`](./supergraphs#configuration).
```

Comment:
Perhaps it's worth noting what the old config format looks like/the info it contains and then map each of those values to how you'd configure it in rover?

Reply:
I'll see what I can do for that. I had the same thought, but there's a bunch of edge cases with the old config. I think I can figure something out though 😄